platform/darwin/src/MGLShapeSource.h

```diff
@@ -143,6 +142,41 @@ extern const MGLShapeSourceOption MGLShapeSourceOptionSimplificationTolerance;
  */
 - (instancetype)initWithIdentifier:(NSString *)identifier shape:(nullable MGLShape *)shape options:(nullable NS_DICTIONARY_OF(MGLShapeSourceOption, id) *)options NS_DESIGNATED_INITIALIZER;
 
+/**
+ Returns a shape source with an identifier, a shape that is a 
+ `MGLShapeCollectionFeature` holding the features passed in, and dictionary of 
+ options for the source.
+ 
+ Passing an array of `MGLShape` subclass instances that conform to the 
```

Comment:
Maybe this could be a `@note` instead? My thinking is that we should take every chance we have to point out the subtle difference between shapes and features. If we do get rid of `-initWithIdentifier:shapes:options:` then this note could probably go away completely.